projects/ngx-openlayers/src/lib/sources/imagewms.component.ts

```diff
@@ -36,13 +46,26 @@ export class SourceImageWMSComponent extends SourceComponent implements OnChange
   @Input()
   url: string;
 
+  @Output()
+  onImageLoadStart: EventEmitter<source.ImageEvent>;
+  @Output()
+  onImageLoadEnd: EventEmitter<source.ImageEvent>;
+  @Output()
+  onImageLoadError: EventEmitter<source.ImageEvent>;
+
   constructor(@Host() layer: LayerImageComponent) {
     super(layer);
+    this.onImageLoadStart = new EventEmitter<source.ImageEvent>();
```

Comment:
As for image static, we should initialize those directly in the field

Reply:
ouch, forgot this one, sorry...